### PR TITLE
add s3 output plugin to list

### PR DESCRIPTION
**Pull request by caleb15**
recently got added - see https://github.com/fluent/fluent-bit/issues/1004
I also capitalized the start of some lines for consistency.

----
Enter `[N/A]` in the box, if an item is not applicable to your change.

**Testing**
Before we can approve your change; please submit the following in a comment:
- [ ] Example configuration file for the change
- [ ] Debug log output from testing the change

- [ ] Attached [Valgrind](https://valgrind.org/docs/manual/quick-start.html) output that shows no leaks or memory corruption was found

**Documentation**

- [ ] Documentation required for this feature

----

Fluent Bit is licensed under Apache 2.0, by submitting this pull request I understand that this code will be released under the terms of that license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,12 +146,13 @@ Fluent Bit is fully supported on Windows environments, get started with [these i
 | [influxdb](https://docs.fluentbit.io/manual/pipeline/outputs/influxdb) | InfluxDB | Flush records to InfluxDB time series database. |
 | [kafka](https://docs.fluentbit.io/manual/pipeline/outputs/kafka) | Apache Kafka | Flush records to Apache Kafka |
 | [kafka-rest](https://docs.fluentbit.io/manual/pipeline/outputs/kafka-rest-proxy) | Kafka REST Proxy | Flush records to a Kafka REST Proxy server. |
-| [nats](https://docs.fluentbit.io/manual/pipeline/outputs/nats) | NATS | flush records to a NATS server. |
-| [null](https://docs.fluentbit.io/manual/pipeline/outputs/null) | NULL | throw away events. |
+| [nats](https://docs.fluentbit.io/manual/pipeline/outputs/nats) | NATS | Flush records to a NATS server. |
+| [null](https://docs.fluentbit.io/manual/pipeline/outputs/null) | NULL | Throw away events. |
+| [s3](https://docs.fluentbit.io/manual/pipeline/outputs/s3) | NULL | Flush records to s3 |
```
Comment (PettitWesley) [inline README.md:151]: Title should be S3 not Null?

**Comment by PettitWesley** [comment]
You need to commit your changes with the `--sign` option

**Comment by PettitWesley** [comment]
Also your commit message does not fit the requirements listed in the Contributing.md

File: README.md (hunk 1 above)
Comment (caleb15) [reply README.md:151]: The title is S3. The null you're looking at is on the above line. The above line shows as changed because I capitalized "throw"

**Comment by caleb15** [comment]
> You need to commit your changes with the --sign option

image

Github says the commit is signed.

**Comment by caleb15** [comment]
oh, it's signed but it's not _signed off_, I see

File: README.md (hunk 1 above)
Comment (PettitWesley) [reply README.md:151]: Screen Shot 2020-11-11 at 2 56 27 PM

I see the title as NULL

**Comment by PettitWesley** [comment]
DCO is still failing- you need to re commit with the `--sign` option.

File: README.md (hunk 1 above)
Comment (caleb15) [reply README.md:151]: Oh, I was looking at the name, my bad.

**Review by PettitWesley — APPROVED** [review]

**Comment by PettitWesley** [comment]
Thanks!

**Comment by PettitWesley** [comment]
> Also your commit message does not fit the requirements listed in the Contributing.md

Oh I forgot to check this again....


@edsiper Apologies, I merged this one without checking the commit message again 😐

**Pull request merged**